neurovault/apps/statmaps/views.py

```diff
@@ -697,9 +697,13 @@ def compare_images(request,pk1,pk2):
     atlas = pybrainatlas.atlas(atlas_xml,atlas_file)
 
     # Create custom image names and links for the visualization
+    image1_custom = "%s : %s" %(image1.name,image1.collection.name)
+    image2_custom = "%s : %s" %(image2.name,image2.collection.name)
+    if len(image1_custom) > 125: image1_custom = "%s..." %image1_custom[0:125] 
```

Comment:
Could we truncate image name and collection name separately? This way at least a snippet of both will be always visible.